src/main/webapp/app/core/course/overview/exercise-details/request-feedback-button/request-feedback-button.component.html

```
@@ -1,4 +1,4 @@
-@if (!isExamExercise && requestFeedbackEnabled) {
+@if (!isExamExercise) {
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)